test-suite/misc/4722.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+
+if ! test -d misc/4722; then
```

Comment:
I don't like this test for some reason. Would you mind doing 
```bash
mkdir -p misc/4722
ln -sfT toto misc/4722/tata
touch misc/4722.v
```
instead?

Reply:
*shrug* I find that entirely unreadable and much less clear than my code, but I don't care strongly.